### DISCUSSION TITLE
2022-01-17 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#180] [Core]
# TL;DR
Q1 OKRs planned with main 2 goals:
* test scaling of the system to 4Petagas/second
* lower the cost of storage

# Done

@mzhangmzz - Fixed a bug in receipts ordering https://github.com/near/nearcore/pull/6062
@mzhangmzz - A document for testing analysis (only partly done) 
@mzhangmzz - perf reviews
@mzhangmzz - fix testnet chunk requesting errors
@mzhangmzz - learn how to use mocknet and add grafana graph

@EgorKulikov - perf reviews
@EgorKulikov - learning how to run python tests locally

@Longarithm - first PR to introduce rocksdb metrics
@Longarithm - rocksdb learning of the internals
@Longarithm - trie iteration performance debugging

@mm-near - OKR planning and Aurora discussions on how to improve their costs
@mm-near - finished adding documentation to remaining database columns (https://github.com/near/nearcore/pull/6068))
@mm-near - learning about internals of rocksdb
